.pre-commit-config.yaml

```diff
@@ -0,0 +1,44 @@
+
```

Comment:
FYI you may have merge conflicts with #69

The empty line at the top of the file was causing merge conflicts when I was accidentally trying to merge this into `fix/move_group`. (It was a mistake because I needed to merge `feauture/planning_scene_ros_api_tutorial` into `fix/move_group` )

Reply:
Oh I see will fix it.

Reply:
Thanks a lot!